owncloudApp/build.gradle

```diff
@@ -22,7 +22,7 @@ dependencies {
     implementation "androidx.appcompat:appcompat:$appCompat"
     implementation 'com.getbase:floatingactionbutton:1.10.1'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-    implementation "androidx.browser:browser:$androidX"
+    implementation "androidx.browser:browser:1.2.0"
```

Comment:
I think this is not related with Android 3.5.3, is it? Next time instead of replacing with a value, create a new variable in the build gradle as it is kotlinVersion.

Reply:
It's not related.
A variable makes only sense, if you use it at least twice. Here it isn't.
Using everywhere a variable (without the need) only removes the ability to update with 
intention action

<img width="527" alt="image" src="https://user-images.githubusercontent.com/3314607/72161442-69ccfa80-33c0-11ea-9bd1-dbca9c6e46d0.png">